src/FSharpx.Extras/ComputationExpressions/Validation.fs

```diff
@@ -1,5 +1,6 @@
 ﻿namespace FSharpx
 
+[<System.Obsolete("This module is deprecated. Use Validation.Result instead.")>]
 module Validation =
```

Comment:
Note: During review check if the obsolete warning works as expected (bugs old code users and leaves new code users alone).

Reply:
Checked, works as expected.